nixos/modules/system/activation/activation-script.nix

```diff
@@ -137,8 +140,13 @@ in
 
         mkdir -m 1777 -p /var/tmp
 
+        # Make sure it's really empty
+        chattr -i /var/empty
+        rm -rf /var/empty
```

Comment:
This introduces a race during activation where `/var/empty` doesn't exist. So for example, if it gets interrupted at that point, you won't be able to log in via ssh anymore.